client_with_ssl.go

```diff
@@ -8,420 +24,208 @@ import (
 	config.Cluster.Cloud.Token = "YOUR_CLUSTER_DISCOVERY_TOKEN"
 	config.Stats.Enabled = true
 	config.Stats.Period = types.Duration(time.Second)
+
 	caFile, err := filepath.Abs("./ca.pem")
 	if err != nil {
 		panic(err)
 	}
+
 	certFile, err := filepath.Abs("./cert.pem")
 	if err != nil {
 		panic(err)
 	}
+
 	keyFile, err := filepath.Abs("./key.pem")
 	if err != nil {
 		panic(err)
 	}
+
 	config.Cluster.Network.SSL.Enabled = true
 	config.Cluster.Network.SSL.SetTLSConfig(&tls.Config{ServerName: "hazelcast.cloud"})
+
 	err = config.Cluster.Network.SSL.SetCAPath(caFile)
 	if err != nil {
 		panic(err)
 	}
+
 	err = config.Cluster.Network.SSL.AddClientCertAndEncryptedKeyPath(certFile, keyFile, "YOUR_SSL_PASSWORD")
 	if err != nil {
 		panic(err)
 	}
+	config.Serialization.Compact.SetSerializers(&CitySerializer{})
+
 	client, err := hazelcast.StartNewClientWithConfig(ctx, config)
 	if err != nil {
 		panic(err)
 	}
 	defer client.Shutdown(ctx)
 	fmt.Println("Connection Successful!")
 
-	// Uncomment the example you want to run.
-	mapExample(client)
-	// sqlExample(client)
-	// jsonSerializationExample(client)
-	// nonStopMapExample(client)
-}
-
-func city(country string, name string, population int) serialization.JSON {
-	text := fmt.Sprintf(`{
-		"country":"%s", 
-		"city": "%s", 
-		"population":"%d"
-	}`, country, name, population)
-	return serialization.JSON(text)
-
-}
-
-func country(isoCode string, name string) serialization.JSON {
-	text := fmt.Sprintf(`{
-		"isoCode":"%s", 
-		"country": "%s"
-	}`, isoCode, name)
-	return serialization.JSON(text)
+	createMapping(client)
+	populateCities(client)
+	fetchCities(client)
 }
 
-func mustPut(_ interface{}, err error) {
+func createMapping(client *hazelcast.Client) {
+	// See: https://docs.hazelcast.com/hazelcast/latest/sql/mapping-to-maps
+	fmt.Print("\nCreating the mapping...")
+	mappingQuery := fmt.Sprintf(`
+		 CREATE OR REPLACE MAPPING 
+		 cities (
+			__key INT,                                        
+			country VARCHAR,
+			city VARCHAR,
+			population INT) TYPE IMAP
+		 OPTIONS ( 
+			'keyFormat' = 'int',
+			'valueFormat' = 'compact',
+			'valueCompactTypeName' = 'city')`)
+	_, err := client.SQL().Execute(context.Background(), mappingQuery)
 	if err != nil {
 		panic(err)
 	}
+	fmt.Print("OK.")
 }
 
-func mustGet(value interface{}, err error) interface{} {
-	if err != nil {
-		panic(err)
-	}
-	return value
-}
+func populateCities(client *hazelcast.Client) {
+	fmt.Print("\nInserting data via SQL...")
+	insertQuery := fmt.Sprintf(`INSERT INTO cities 
+										(__key, city, country, population) VALUES
+										(1, 'London', 'United Kingdom', 9540576),
+										(2, 'Manchester', 'United Kingdom', 2770434),
+										(3, 'New York', 'United States', 19223191),
+										(4, 'Los Angeles', 'United States', 3985520),
+										(5, 'Istanbul', 'Türkiye', 15636243),
+										(6, 'Ankara', 'Türkiye', 5309690),
+										(7, 'Sao Paulo ', 'Brazil', 22429800)`)
```

Comment:
This indentation looks like improper.

Reply:
Decerased at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/14/commits/5c7755ff8f3d10b0c43c331020c9bf7aea4d8096